massa-pos-exports/src/lib.rs

```diff
@@ -15,6 +16,7 @@ mod pos_changes;
 mod pos_final_state;
 mod settings;
 
+pub use config::*;
```

Comment:
Why not importing PosConfig directly here?

Reply:
Done